docs/releases/3.0.txt

```diff
@@ -222,7 +222,11 @@ CSRF
 Email
 ~~~~~
 
-* ...
+* The new ``reporter_class`` parameter of
```

Comment:
Not sure if this wouldn't be better off in a `Logging` section...

Reply:
Agreed.